### ISSUE TITLE
is the learning rate of Generator and Discriminator the same?

### ISSUE THREAD

**Issue by khryang** [junyanz/pytorch-CycleGAN-and-pix2pix#154]
I read the paper of CycleGAN that the learning rate of G and D are different.
but in codd of 'train_option.py' from this hub, there is only one value of 'lr'.
and I cannot find any address for this in code of 'cycle_gan.py'

so, I think in this pytorch code, the network use the same learning rate for G and D.
Am I right?

**Comment by junyanz** [comment]
Essentially we [divide](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/cycle_gan_model.py#L115) the objective by 2 while optimizing D, which slows down the rate at which D learns relative to G. We will clarify it in the arxiv v3.

**Comment by CaelynCheung1996** [comment]
I was wondering how to define the learning rate for G and D separately. And if the loss of only one of the generator blows up while the others performs well, how can we change the learning rate according to this problem.
